akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala

```diff
@@ -187,10 +188,16 @@ private[http] class HttpRequestParser(_settings: ParserSettings,
           if (te.isChunked) {
             if (clh.isEmpty) {
               emitRequestStart(chunkedEntity(cth, expect100continueHandling), completedHeaders)
-              parseChunk(input, bodyStart, closeAfterResponseCompletion)
+              parseChunk(input, bodyStart, closeAfterResponseCompletion, totalBytesRead = 0L)
```

Comment:
I agree. It boils down to taking a bit more care on the inside vs. making it easier to use from the outside. If it's used solely locally I'd also vote against defaults.